content/blog/the-understanding-urbit-podcast.md

```diff
@@ -0,0 +1,26 @@
++++
+title = "The Understanding Urbit Podcast"
+date = 2020-04-03
+description = "An interview-based podcast series about the Urbit project, as told by those working on it."
+[extra]
+author = "Arthur Falls"
+ship = "~satsyt-sogleb"
+image ="https://media.urbit.org/site/posts/essays/uu-podcast.png"
++++
+
+<br>
+
+<img class="ba" src="https://media.urbit.org/site/posts/essays/uu-podcast.png">
+
+The Understanding Urbit Podcast
+~satsyt-sogleb
```

Comment:
Cut this, as we already have top matter above this

Reply:
Good catch